frontend/src/App.jsx

```diff
@@ -29,6 +29,7 @@ function App() {
         <VideoSlider slideNumber={4} videos={videos} />
         <Routes>
           <Route path="/category/:categoryName" element={<CategoryPage />} />
+          <Route path="/searchResults/:query" element={<SearchResults />} />
```

Comment:
urls should be in kebab-case : `/search` is enough, and you should not use params for a query